scripts/write-config.sh

```diff
@@ -297,6 +297,8 @@ workspace:
     managed-app-tenant-id: ${tenantid}
     sas-token-start-time-minutes-offset: 15
     sas-token-expiry-time-minutes-offset: 60
+feature:
```

Comment:
Does this work? The include of `local-properties.yml` is in `application.yml`, so there will be two definitions of `tps-enabled`.  I don't know which one wins.

Reply:
Yep! Imports are treated as additional documents appended to the end of the importing document ([Spring doc](https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.external-config.files.importing)) so the values in `local-properties` are guaranteed to override the values in `application.yml`. I've also verified this works locally.

Added a comment above the import clarifying this as well since it's important behavior